components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt

```diff
@@ -133,6 +133,8 @@ internal class GeckoPromptDelegate(private val geckoEngineSession: GeckoEngineSe
 
         val isMultipleFilesSelection = selectionType == GeckoSession.PromptDelegate.FILE_TYPE_MULTIPLE
 
+        val captureMode = PromptRequest.File.FacingMode.NONE
```

Comment:
GeckoView doesn't support `capture` _yet_ but will soon. I'm adding the Android Components side for now. Eventually we'll receive this attribute from GV.

I added this line so that I can deprecate the old constructor.

Reply:
Also added WebView support to show how it might be used by GeckoView later 😄